owl/EASE-STATE.owl

```diff
@@ -73,41 +74,27 @@
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#Accessor -->
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Container -->
 
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#Accessor">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#UsuallyAgentive"/>
-    </owl:Class>
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Container"/>
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#AssemblyConnection -->
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#LinkedObject -->
 
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#AssemblyConnection">
```

Comment:
for now we only have the Linkage* concepts and do not refer to *assembly*

Reply:
As pointed out in our meeting, the concept would move to either knowrob.owl or knowrob_assembly.owl. The rational is that assembly is not an everyday activity.